### PR TITLE
Few updates

### PR DESCRIPTION
**Pull request by GhasBarbouchi**
.update the icon and fix descreption
.change the period unity from seconds to milliseconds

### CODE REVIEW

File: src/module/module.py [hunk 1]
```diff
@@ -46,4 +46,4 @@ def module_main():
                 log.error(send_error)
             else:
                 log.debug("Data sent sucessfully.")
-        sleep(int(getenv('PERIOD')))
+        sleep(int(getenv('PERIOD'))/1000)
```
Comment (europaul) [inline src/module/module.py]: please use named constants

File: Module.yaml [hunk 2]
```diff
@@ -27,34 +25,23 @@ envs:                                   # Array of environment variables that wi
...
-    description: Offset of a value (used in case a sensor provide multiple values)     # Description/helper text for the field.
+    description: Offset in case the sensor has more than one registers available (in decimal)       # Description/helper text for the field.
     value: 0                            # Value to be set in the environment variables.
     type: number                        # Type of the field. Possible values are: ['text', 'select', 'boolean', 'date', 'time', 'dateTime-local', 'number', 'multiple', 'CSV', 'file']
-    options: []                         # Options for the field if it is a select or multiple select. An array of string options
-    dependencies: []                    # List of dependencies for the field. This field is only displayed if all the dependencies are met. If
-ports:                                  # Array of ports that will be exposed by the module.
-  - container: 8080                     # Port number from the container.
-    host: 8080                          # Port number to map on the host.
-tags:
```
Comment (europaul) [inline Module.yaml:46]: did you accidentally delete this line?

File: Module.yaml [hunk 3]
```diff
@@ -27,34 +25,23 @@ envs:                                   # Array of environment variables that wi
...
+  - name: Polling Period               # name of the field to be displayed on the UI.
+    key: PERIOD                          # key to set in the environment variables.
+    description: The I2C interface read data every choosen period (in milliseconds)       # Description/helper text for the field.
+    value: ''                           # Value to be set in the environment variables.
```
Comment (europaul) [inline Module.yaml]: there is no default value for polling period?

File: example.env [hunk 4]
```diff
@@ -4,7 +4,7 @@ MODULE_NAME=i2c-input
 MODULE_TYPE=Input
 I2C_INTERFACE_NUMBER=1
 DATA_TYPE=byte
-SLAVE_ADDR=10
+SLAVE_ADDR=104
 PERIOD=1
```
Comment (europaul) [inline example.env]: this needs to be changed to milliseconds I assume, also in other places where you define default values

File: Module.yaml [hunk 5]
```diff
@@ -25,27 +25,24 @@ envs:                                   # Array of environment variables that wi
     options:
       - byte
       - word
-  - name: I2C slave address (in decimal) # name of the field to be displayed on the UI.
+  - name: I2C slave address  # name of the field to be displayed on the UI.
     key: SLAVE_ADDR                      # key to set in the environment variables.
-    description: I2c slave address       # Description/helper text for the field.
+    description: I2c slave address (in decimal)     # Description/helper text for the field.
```
Comment (europaul) [inline Module.yaml]:
```suggestion
    description: I2C slave address (in decimal)     # Description/helper text for the field.
```

**Review by europaul — CHANGES_REQUESTED** [review]

File: Module.yaml (hunk 2 above)
Comment (GhasBarbouchi) [reply Module.yaml:46]: nope, I thought it doesn't matter to keep it because there is no mounting device anymore.

File: Module.yaml (hunk 3 above)
Comment (GhasBarbouchi) [reply Module.yaml]: yes, not a default value but to make it easier for the user we can put a random value.

File: example.env (hunk 4 above)
Comment (GhasBarbouchi) [reply example.env]: the period could be arbitrarily 1000ms (that is why no matter to use that value as default value)

Comment (europaul) [reply example.env]: but rn your default value in 1ms. do you want to wait for 1ms?

File: Module.yaml (hunk 3 above)
Comment (europaul) [reply Module.yaml]: this is called the default value then :wink:

File: Module.yaml (hunk 2 above)
Comment (europaul) [reply Module.yaml:46]: I'm talking about `tags:`

**Review by europaul — APPROVED** [review]

**Pull request merged**